pkg/application/prompt.go

```diff
@@ -95,8 +95,12 @@ func getDescription(t *template.Data) error {
 	return nil
 }
 
+var IgnoredTemplateDirs = []string{"alpha"}
```

Comment:
the choice of list of string here makes the code that uses this value less performant / readable (see comment below)
```suggestion
var IgnoredTemplateDirs = map[string]bool{"alpha": true}
```